README.md

```diff
@@ -46,7 +46,10 @@ UI Refactor(Including Audio Player) -> Add Travis -> Connect a DataBase(FireBase
 * To run the server, use the command `pipenv run python3 server.py`
 
 ### To start the React Frontend
-* Clone the repo using `git clone https://github.com/NJACKWinterOfCode/youtube_cli_flask.git`.
+
+* Open terminal to get started.
+
+* To clone the repo, type `git clone https://github.com/NJACKWinterOfCode/youtube_cli_flask.git` in your terminal.
```

Comment:
This step is not required now. Can you please delete it?

Reply:
so I should remove the react part? is that what you mean? I do not understand what you're saying. Sorry for the inconvenice.

Reply:
@LiteBrick204 , just change this url `https://github.com/NJACKWinterOfCode/youtube_cli_flask.git` to the repo url. 😄